drivers/pinctrl/pinctrl_npcx.c

```diff
@@ -167,8 +167,6 @@ int pinctrl_configure_pins(const pinctrl_soc_pin_t *pins, uint8_t pin_cnt,
 		} else if (pins[i].flags.type == NPCX_PINCTRL_TYPE_PSL_IN) {
 			/* Configure SPL input's detection mode */
 			npcx_psl_input_detection_configure(&pins[i]);
-		} else {
-			return -ENOTSUP;
```

Comment:
Is dropping this intentional? (was introduced in the previous patch...)

Reply:
Yes, thanks for reminding this.

Reply:
Done.